css/main.css

```diff
@@ -0,0 +1,406 @@
+  -o-linear-gradient(linear, 0 100%, 0 0, from(#666),
+  color-stop(0.5, #5e5e5e),
+  color-stop(0.51, #707070),
+  to(#808080));
+  background: -ms-linear-gradient(linear, 0 0, 0 100%,
+  from(rgba(0, 0, 0, .2)),
+  color-stop(0.05, rgba(0, 0, 0, 0)),
+  color-stop(0.97, rgba(0, 0, 0, 0)),
+  to(rgba(0, 0, 0, .45))),
+  -ms-linear-gradient(linear, 0 0, 100% 0,
+  from(rgba(0, 0, 0, .2)),
+  color-stop(0.002, rgba(0, 0, 0, 0)),
+  color-stop(0.998, rgba(0, 0, 0, 0)),
+  to(rgba(0, 0, 0, .2))),
+  url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%,
+  -ms-linear-gradient(linear, 0 100%, 0 0, from(#666),
+  color-stop(0.5, #5e5e5e),
+  color-stop(0.51, #707070),
+  to(#808080));
+  background: -webkit-linear-gradient(linear, 0 0, 0 100%,
+  from(rgba(0, 0, 0, .2)),
+  color-stop(0.05, rgba(0, 0, 0, 0)),
+  color-stop(0.97, rgba(0, 0, 0, 0)),
+  to(rgba(0, 0, 0, .45))),
+  -webkit-linear-gradient(linear, 0 0, 100% 0,
+  from(rgba(0, 0, 0, .2)),
+  color-stop(0.002, rgba(0, 0, 0, 0)),
+  color-stop(0.998, rgba(0, 0, 0, 0)),
+  to(rgba(0, 0, 0, .2))),
+  url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%,
+  -webkit-linear-gradient(linear, 0 100%, 0 0, from(#666),
+  color-stop(0.5, #5e5e5e),
+  color-stop(0.51, #707070),
+  to(#808080));
+  height: 36px;
+  position: relative;
+  top: 4.6rem;
+  left: -2%;
+  margin: 0 auto;
+  z-index: 900;
+  border-radius: 4px;
+  width: 845px;
+
+}
+
+.navigation-ul-list {
+  display: inline-block;
+  position: relative;
+  width: 100px;
+  height: 35px;
+  text-align: center;
+}
+
+
+.navigation-link:link,
+.navigation-link:visited{
+text-decoration: none;
+color: white;
+}
+
+.navigation-ul-list:not(:last-child) {
+  border-width: 0 2px 0 1px;
+  /* border-top-width: 0px; */
+  border-right-width: 2px;
+  /* border-bottom-width: 0px; */
+  border-left-width: 1px;
+  border-style: solid;
+  border-top-style: solid;
+  border-right-style: solid;
+  border-bottom-style: solid;
+  border-left-style: solid;
+  -o-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -ms-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -moz-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -webkit-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+}
+
+.navigation-ul-list:hover:not(:last-child) {
+  background: #575757;
```

Comment:
since it's only color you're setting here `background-color` is fine. Could even be set in line below